tests/test_magma.py

```diff
@@ -0,0 +1,105 @@
+from peak import Peak, family_closure, assemble, Enum_fc
```

Comment:
Need tests for new style  that match the typical structure.
`isa.py`
`sim.py` 
`test.py`

Reply:
Also needs tests where sim1.py imports component from sim2.py

Reply:
Still need above

Reply:
This test exists as test_composition. What do you think I am missing here?

Reply:
Ahh sorry missed that